meta-hovergames-distro/recipes-core/udev/udev-rules-imx/10-imx.rules

```diff
@@ -0,0 +1,28 @@
+# Create symlinks for i.mx keypads and touchscreens
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name}=="mxckpd",     SYMLINK+="input/keyboard0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name}=="mxc_ts",     SYMLINK+="input/ts0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name}=="imx_adc_ts", SYMLINK+="input/ts0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name}=="mpr084",     SYMLINK+="input/keyboard0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name}=="tsc2007",    SYMLINK+="input/ts0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name} =="STMP3XXX touchscreen",    SYMLINK+="input/ts0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name} =="MXS touchscreen",    SYMLINK+="input/ts0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name} =="HannStar P1003 Touchscreen",	SYMLINK+="input/ts0"
+#SUBSYSTEM=="input" KERNEL=="event*" ATTRS{name} =="eGalax Touch Screen",	SYMLINK+="input/ts0"
+
+# The long class name gets cut off to be mc13783_connectiv in 
+# /sys/class/mc13783_connectivity/mc13783_connectivit
+KERNEL=="mc13783_connectiv*",  NAME="mc13783_connectivity"
+# Anyone has readonly permission to IIM device file
+KERNEL=="mxc_iim",  MODE="0444", SYMLINK+="mxc_mem"
+KERNEL=="mxs_viim", MODE="0444", SYMLINK+="mxc_mem"
+KERNEL=="mxc_ipu",  MODE="0600"
+KERNEL=="mxc_vpu",  MODE="0600"
+SUBSYSTEM=="video", MODE="0660"
+KERNEL=="fb[0-9]",  MODE="0660", GROUP="video"
+KERNEL=="gsl_kmod", MODE="0660", GROUP="video"
+KERNEL=="galcore",  MODE="0660", GROUP="video"
+
+# Fix for SDMA firmware not loading for two out of 3 dma controllers on imx8mm
+SUBSYSTEM=="platform" KERNEL=="302b0000.dma-controller", ACTION=="add", RUN+="/etc/sdma"
+SUBSYSTEM=="platform" KERNEL=="302c0000.dma-controller", ACTION=="add", RUN+="/etc/sdma"
+SUBSYSTEM=="platform" KERNEL=="30bd0000.dma-controller", ACTION=="add", RUN+="/etc/sdma"
```

Comment:
You don't have to do this here, but generally I like to make one commit that copies what is upstream directly then a second commit that shows the change.

Reply:
Understood!